committers.md

```diff
@@ -132,6 +132,27 @@ In particular, if you are working on an area of the codebase you are unfamiliar
 Git history for that code to see who reviewed patches before. You can do this using 
 `git log --format=full <filename>`, by examining the "Commit" field to see who committed each patch.
 
+<h3>When to commit/merge a pull request</h3>
+
+PRs shall not be merged during active, on-topic discussion unless they address issues such as critical security fixes of a public vulnerability. Under extenuating circumstances, PRs may be merged during active, off-topic discussion and the discussion directed to a more appropriate venue. Time should be given prior to merging for those involved with the conversation to explain if they believe they are on-topic.
```

Comment:
In this context, `Time should be given prior to merging`, can we have an example like 72 hour explicitly?

Reply:
Ah, I don't think we can change the wording anymore since we voted on it.

Reply:
Or well we can but I'd prefer to take it to another vote first before changing the policy.

Reply:
I think minor clarifications are entirely fine. No need for a new vote.

Reply:
Sounds good to me then. So we can do something like "often at least 72 hours"?

Reply:
+1 for `often at least 72 hours`

Reply:
I think we can just say for example 72 hours. That's [the example in the lazy consensus](https://www.apache.org/foundation/voting.html#LazyConsensus), and I believe we're rather clarifying and reiterate what we wrote there here.